src/scss/includes/panels/favorite_panel.scss

```diff
@@ -148,41 +125,29 @@
 }
 
 .favorite_panel__container__title {
-  color: $text_color;
-  padding-top: 10px;
-  margin: 20px 0 -4px 20px;
+  margin-bottom: 10px;
+  padding: 14px 20px;
+  font-size: 12px;
   font-weight: normal;
-  font-size: 22px;
+  color: #5c6f84;
+  border-bottom: solid 1px #f4f6fa;
 }
 
 .favorite_panel__item__desc {
   font-size: 12px;
   color : #8d929c;
 }
 
-.favorite_panel__swipe_window {
-  clear: both;
-}
-
 .favorite_panel__item__more_container {
   position: relative;
-}
-
-.favorite_panel__swipe_element {
-  position: relative;
-  overflow: hidden;
-  width: $panel_width;
-  height: 70px;
-  padding: 0 8px 0 4px;
-  transition: height .2s;
-  cursor: pointer;
```

Comment:
We lost the "pointer" on favorite elements :cry:

Reply:
fixed on 431